utils/polus-bfio-util/bfio/bfio.py

```diff
@@ -375,171 +621,361 @@ def _init_writer(self):
         """
         jutil.run_script(script,
                          dict(path=self._file_path,
-                              xml=self._metadata.to_xml().replace('<ome:','<').replace('</ome:','</'),
+                              xml=self._metadata.to_xml().replace('<ome:', '<').replace('</ome:', '</'),
                               writer=writer))
         writer.setId(self._file_path)
         writer.setInterleaved(False)
         writer.setCompression("LZW")
         x = writer.setTileSizeX(self._TILE_SIZE)
         y = writer.setTileSizeY(self._TILE_SIZE)
-        
-        self._pix['chunk'] = self._MAX_BYTES/(self._pix['spp']*self._pix['bpp'])      # number of pixels to load at a time
-            
+
+        # number of pixels to load at a time
+        self._pix['chunk'] = self._MAX_BYTES / \
+            (self._pix['spp']*self._pix['bpp'])
+
         self.__writer = writer
+
+    def pixel_type(self, dtype=None):
+        """pixel_type Get/Set the pixel type
+
+        If dtype is not defined, this function returns the current file
+        setting. If dtype is defined, it must be one of the following:
         
-    def pixel_type(self,dtype=None):
+        'uint8':  Unsigned 8-bit pixel type
+        'int8':   Signed 8-bit pixel type
+        'uint16': Unsigned 8-bit pixel type
+        'int16':  Signed 16-bit pixel type
+        'uint32': Unsigned 32-bit pixel type
+        'int32':  Signed 32-bit pixel type
+        'float':  IEEE single-precision pixel type
+        'double': IEEE double precision pixel type
```

Comment:
do we also want to support complex (2x double)? I feel like the discussion with Mahdi made me think about this the other day and I think you were right to say perhaps in the future we should support it... if we do want to do that, today is that future :)

Reply:
Do I want to? No. Do we have a need to right now? I don't think so. Until we have a plugin that might export complex pixel type images (such as an FFT), I don't see the need to do it.

This PR is just about documentation.